libcore/StockDialogs.vala

```diff
@@ -88,7 +101,12 @@ namespace PF.Dialogs {
         } else {
             var response_id = 0;
             foreach (string title in data.button_titles) {
-                dialog.add_button (title, response_id++);
+                dialog.add_button (title, response_id);
+                if (title == DELETE || title == DELETE_ALL) {
+                    var button = dialog.get_widget_for_response (response_id);
+                    button.get_style_context ().add_class ("destructive-action");
```

Comment:
It's better to use the constants for these so we get deprecation warnings in case they ever change or are removed :)

```suggestion
                    button.get_style_context ().add_class (Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
```

Reply:
Good to know, thanks! I added that.